angular-client/src/app/pages/shared/view-appointment-dialog/view-appointment-dialog.component.scss

```diff
@@ -0,0 +1,21 @@
+:host{
+  --icon-size: 2rem;
+
+  #dialog-container{
+    position: relative;
+  }
+
+  #close-button{
+    .mat-icon{
+      font-size: var(--icon-size);
```

Comment:
why are you using var(--...) here, when you could use $variableName? SCSS allows stuff like that